packages/chaire-lib-common/src/config/osm/overpassQueries/streets.overpassql

```diff
@@ -0,0 +1,8 @@
+
+[out:json][timeout:25];
```

Comment:
What's the rationale for a 25 secondes timeout ?

Reply:
I don't know, what would you put there?

Reply:
added a comment and changed to 10 seconds